client/components/admin/AdminTeams/imports/AdminTeamUserList.jsx

```diff
@@ -0,0 +1,101 @@
+import { Meteor } from 'meteor/meteor'
+import { withTracker } from 'meteor/react-meteor-data';
+import React, { Component } from 'react';
+import { Table, Icon, Label, Button } from 'semantic-ui-react'
+
+AdminTeamUserList = class AdminTeamsUserList extends Component {
+    constructor(props){
+        super(props);
+
+        this._userRow = this._userRow.bind(this);
+    }
+
+    removeUserFromTeam(userId){
+        Meteor.call('admin.user.setTeam', userId, "", error => {
+            if(error){
+                console.log(error);
+                alert(`Failed to remove user from team. ${error.reason}`);
+            }
+        });
+    }
+
+    _userRow(user){
+        const { checkedIn } = user;
+        const isOwner = this.props.owner == user._id;
+        return (
+            <Table.Row key={user.email}>
+                <Table.Cell>
+                    {isOwner ?
+                        <Label ribbon>
+                            <Icon name="trophy" color="orange" />
+                            Owner
```

Comment:
This file also has 4 space or tab indentation

Reply:
same as above